ccloud-monitoring/docs/consumer-scenarios/increasing-consumer-lag.rst

```diff
@@ -0,0 +1,168 @@
+.. _ccloud-monitoring-consumer-increasing-consumer-lag:
+
+Increasing Consumer Lag
+***********************
+
+Consumer lag is a tremendous performance indicator. It tells you the offset difference between the
+producer's last produced message and the consumer group's last commit. If you are unfamiliar with
+consumer groups or concepts like committing offsets, please refer to this
+`Kafka Consumer documentation <https://docs.confluent.io/platform/current/clients/consumer.html>`__.
+
+A large consumer lag, or a quickly growing lag, indicates that the consumer is not able to keep up with
+the volume of messages on a topic.
+
+This scenario will look at |ccloud| metrics from the Metrics API, `kafka-lag-exporter <https://github.com/lightbend/kafka-lag-exporter>`__ metrics, and
+client metrics from the client application’s MBean object ``kafka.consumer:type=consumer-fetch-manager-metrics,client-id=<client_id>``.
+
+Introduce failure scenario
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+#. By default 2 consumers and 1 producer are running. Adjust that to 1 consumer and 5 producers in order to show an increase in consumer lag.
```

Comment:
Just curious, why is the default 2 consumers?  Why isn't it 1 consumer?  If each consumer-related scenario can scale the number of consumers as it needs to, would it make sense (for footprint and ease of explaining) to make the default 1 consumer?

Reply:
Great point. Now that the docker-compose has clients in state to scale with an easy command, I should probably remove the extra consumer. 
Moments like these are why I love PRs.

Reply:
:sonic_waiting: I need to retake screenshots of the scenario, actively working on it.

Reply:
updated the screenshots in https://github.com/confluentinc/examples/pull/948/commits/9b6e040d6ad2773e51a2e0a49e50b944549ac902